log/scope.go

```diff
@@ -100,27 +144,30 @@ func Scopes() map[string]*Scope {
 }
 
 // Fatal outputs a message at fatal level.
-func (s *Scope) Fatal(msg string, fields ...zapcore.Field) {
+func (s *Scope) Fatal(fields ...interface{}) {
 	if s.GetOutputLevel() >= FatalLevel {
-		s.emit(zapcore.FatalLevel, s.GetStackTraceLevel() >= FatalLevel, msg, fields)
+		ie, firstIdx := getErrorStruct(fields)
+		s.callHandlers(FatalLevel, s, ie, fields[0].(string), toZapSlice(firstIdx+1, fields))
 	}
 }
 
 // Fatala uses fmt.Sprint to construct and log a message at fatal level.
 func (s *Scope) Fatala(args ...interface{}) {
```

Comment:
I am not sure how this differs from `Fatal` in terms of output. I think with your interface changes it may actually be identical? Can we verify and if so as a followup remove the `*a` ones?

Reply:
Fatala doesn't take structured parameters to pass to zap. If no one uses those then we can remove (actually I'd replace Fatal with the Fatala implementation and remove Fatala).